test/actor_test.py

```diff
@@ -1906,6 +1906,36 @@ def method(self):
         # we should also test this from a different driver.
         ray.get(new_f.method.remote())
 
+    def testRegisterAndGetActorHandle(self):
+        # TODO(heyucongtom): We may want to test this from another driver?
+        # One viable way might be setting up another ray process here, connecting to the
+        # redis_address, sending the objectID back, and compare
+
+        ray.worker.init(num_workers=1)
+
+        @ray.remote
+        class Foo(object):
+            def method(self):
+                pass
+
+        f1 = Foo.remote()
+        # Test saving f
+        ray.experimental.register_actor("f1", f1)
+        # Test getting f
+        f2 = ray.experimental.get_actor("f1")
+        self.assertEqual(f1._actor_id, f2._actor_id)
+
```

Comment:
Let's modify the actor to be something like

```
def method(self):
    self.x += 1
    return self.x
```

so that here we can do

```
self.assertEqual(ray.get(f1.method.remote(), 1)
self.assertEqual(ray.get(f2.method.remote(), 2)
self.assertEqual(ray.get(f1.method.remote(), 3)
self.assertEqual(ray.get(f2.method.remote(), 4)
```

Reply:
Handled.